core/src/main/java/io/ddf/content/Schema.java

```diff
@@ -597,7 +527,6 @@ public static ColumnClass get(ColumnType type) {
 
   public static class DummyCoding implements Serializable {
```

Comment:
Not the topic of this PR, but we should move DummyCoding out of Schema class into a Pipeline Stage.